daemon/cmd/daemon_main.go

```diff
@@ -637,6 +637,9 @@ func initializeFlags() {
 	flags.Bool(option.EnableSessionAffinity, false, "Enable support for service session affinity")
 	option.BindEnv(option.EnableSessionAffinity)
 
+	flags.Bool(option.EnableServiceTopology, false, "Enable support for service topology aware hints")
```

Comment:
Off by default sounds good as a starting point. Once it's fully GA in K8s, we could also flip it to default on for the agent.

Reply:
One small item, could also be followup from my pov: would be nice to add this to `cilium status --verbose` that it's enabled.

Reply:
I will do it in follow up!